.markdownlint.json

```diff
@@ -0,0 +1,56 @@
+{
+   "default": true,
+   "header-increment": true,
+   "heading-style": {
+      "style": "atx"
+   },
+   "ul-style": {
+      "style": "asterisk"
+   },
+   "list-indent": true,
+   "ul-start-left": false,
+   "ul-indent": {
+      "indent": 3,
+      "start_indented": true
+   },
+   "no-trailing-spaces": true,
+   "no-hard-tabs": true,
+   "no-reversed-links": true,
+   "no-multiple-blanks": { "maximum": 2 },
+   "commands-show-output": true,
+   "line-length": {
+      "line_length": 90,
+      "code_block_line_length": 130
+   },
+   "no-missing-space-atx": true,
+   "no-multiple-space-atx": true,
+   "heading-start-left": true,
+   "no-duplicate-header": {
+      "allow_different_nesting": true
+   },
+   "no-trailing-punctuation":{
+      "punctuation":  ".,;:"
+   },
+   "no-bare-urls": true,
+   "hr-style": { "style": "---" },
+   "no-emphasis-as-heading": false,
+   "no-space-in-emphasis": true,
+   "no-space-in-code": true,
+   "no-space-in-links": true,
+   "fenced-code-language": true,
+   "first-line-heading": false,
```

Comment:
@pbredenberg curious why we would turn this off? Seems like a good rule to have?

Reply:
@jthomerson I think that's my fault. The main reason was that Markdown documents that are a part of a documentation system that turns Markdown into an HTML site typically don't start with headings because they are inserted automatically from YML front-matter. (e.g. all of the MD documents in our Content Styling Documentation project start with either a paragraph or an `###`).

Later, we discovered that markdownlint actually accounted for this case: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md041---first-line-in-file-should-be-a-top-level-heading (the first to last paragraph in that subheading). But, it looks like I never went back and updated the shared document we've been using to keep track of our decisions and reasons.

Reply:
TLDR; it's Matt's fault.